scripts/build_utils.sh

```diff
@@ -13,6 +13,9 @@ function create_venv_dir() {
 function release_from_version() {
     local ver=$1
     case $ver in
+    19.*)
+        rel="squid"
```

Comment:
Should we also drop builds for older distros (e.g. Focal) as part of the kick off?